src/test/resources/org/gradle/profiler/report/example.json

```diff
@@ -1,15 +1,45 @@
 {
   "environment": {
-    "profilerVersion": "0.15.0-alpha2"
+    "profilerVersion": "0.15.0-alpha2",
+    "operatingSystem": "mac os x x86_64 10.15.6",
+    "date": "2020-09-13T12:26:40Z"
   },
   "scenarios": [
     {
       "definition": {
         "name": "no-optimizations",
         "title": "Baseline",
         "displayName": "Baseline using Gradle 6.7-20200903220111+0000",
-        "buildTool": "6.7-20200903220111+0000",
+        "buildTool": "Gradle 6.7-20200903220111+0000",
         "tasks": ":santa-tracker:assembleDebug",
+        "version": "6.7-20200903220111+0000",
+        "gradleHome": "/home/lptr/Workspace/santa-tracker-performance/gradle-user-home/wrapper/dists/gradle-6.7-20200903220111+0000-bin/6csfyoy16renvfjrtt5md548g/gradle-6.7-20200903220111+0000",
+        "javaHome": "/opt/files/jdk-linux/OpenJDK8U-jdk_x64_linux_hotspot_8u242b08.tar.gz",
+        "usesScanPlugin": false,
+        "action": "run tasks :santa-tracker:assembleDebug",
+        "cleanup": "do nothing",
+        "invoker": "Tooling API",
+        "mutator": "ApplyNonAbiChangeToSourceFileMutator(santa-tracker-java/village/src/main/java/com/google/android/apps/santatracker/village/SnowFlake.java)",
```

Comment:
No, this is how it is displayed on the console currently.

Reply:
That is very strange. The `toString()` method of `AbstractFileChangeMutator` uses `sourceFile.toString()`. Looks like we are not passing an absolute file there since `projectDir` is set to `santa-tracker-java`? For the reports generated in the Gradle performance tests, the `sourceFile` will be an absolute path. Do you think that is a problem in the report?

Reply:
I don't think this is a problem at all, but even if we think it is, it's not a problem with how we generate the report. We are just reporting whatever the mutator reports.

This JSON is also only used for a rough estimate of the final result, and the exact path does not really matter here.

Reply:
My worries were that the String in the UI is too long, though I think it is not a problem in the current implementation.